### PR TITLE
Applied a quick fix for "Throwing unknown load command on dataOffsetF……orAddress #39 "

### PR DESCRIPTION
**Pull request by imolowkey**
We received the error "Unknown load command: 0x80000033" even though we passed the old Xcode path as --sdk-root params in our command!

I've added old dylibs files from old Mac OS  to a bundle and read them from the added bundle only for the following  DYLIB files and it's working atm. 🥳

* libsystem_kernel.dylib
* libsystem_platform.dylib
* libsystem_pthread.dylib

*You might still need to add more dylibs files to the bundle and change the code depending on your project requirements. *

**TLDR:**
"Build the project, Copy the ppios-rename and PPiOSBundle.bundle" in the destination project directory or wherever you want and run your command"
**You still need to pass the iPhoneOS.sdk path from the old Xcode (e.g 11-12) as the --sdk-root param **

### CODE REVIEW

**Comment by mobikats** [comment]
"I've added old dylibs files from old Mac OS to a bundle"

Which one please?
I tried the libraries included with Xcode 11.7 but they are still missing at least one.
